custom/scripts/push_images/README.md

```diff
@@ -0,0 +1,441 @@
+## Table of content
+- [Supported Tunables](https://github.com/litmuschaos/test-tools/custom/scripts/push_images#supported-tunables)
```

Comment:
Can we validate the links? 
push_images#supported-tunables or push_images/README.md#supported-tunables

Reply:
Have updated. We can update with the actual links in https://github.com/litmuschaos/test-tools/pull/178 PR.